### ISSUE TITLE
Armory should not show wishlist thumbs on perks

### ISSUE THREAD

**Issue by Prometheus3375** [DestinyItemManager/DIM#10625]
### DIM Version

Version (release), built on 24.06.2024, 05:03:47

### Browser Details

Chrome 126.0.0.0

### OS Details

Windows 10

#### How to reproduce (probably)

1. Open DIM.
2. Click on a weapon with wishlisted roll.
   ![image](https://github.com/DestinyItemManager/DIM/assets/35541026/ef20e669-5c6d-4aa6-94ea-4c0202792d99)
3. Open its weapon card.
   ![image](https://github.com/DestinyItemManager/DIM/assets/35541026/16575a69-ff52-4dea-a2ea-1dc4c7cea619)
4. Close it.
5. Open weapon card of other weapons.
   ![image](https://github.com/DestinyItemManager/DIM/assets/35541026/24ee039d-5775-4dd8-a70d-5b4e99805eb3)
   ![image](https://github.com/DestinyItemManager/DIM/assets/35541026/1e68e0eb-8339-4de4-a3eb-9d787cf58f74)

#### Expected result

No marks on any perk displayed in a weapon card unless a perk is wishlisted.

#### Actual result

I do not wishlist Arrowhead Brake on Yarovit, but it is marked.
I do not have wishlists for Midha at all, but some perks are marked.

**Comment by bhollis** [comment]
I don't actually understand why the perks are showing thumbs on the Armory page at all. That seems to be the bug.

**Comment by Prometheus3375** [comment]
I was thinking the same way, but recently I noticed this is happening quite a lot, so at a time of creating this report I assumed they should display.

**Comment by BasicNullification** [comment]
Since wishlists are viewed top-down anyway, it'd actually be a nice to have feature to have the card highlight the absolute "god-roll" (which is simply the highest ranking roll for that item in the list). I manage my own personal Github wishlist as well as use the default and would like to have the capability to see my top preferred roll at a glance.

To add some emphasis, would even be nice to see top-tier rolls perhaps add a golden outline/overlay/glow/etc when the perks match the top-most listed item on the list exactly.

Maybe this should be its own FR.. _But for all I know_ maybe this was the intention to begin with and perhaps this in itself is what's actually bugged, so it might fit here.

**Comment by bhollis** [comment]
Yeah this is already captured in other issues. See the `Feature: Wish List` label.

**Comment by bhollis** [comment]
The armory perks grid will no longer show wishlist thumbs.
